SendKeptnEventTask/SendKeptnEventTaskV3/index.ts

```diff
@@ -431,6 +435,29 @@ function storeKeptnContext(input: Params, response: any) {
   }
 }
 
+function handleApiError(err: Error | AxiosError) {
+  // If the error is an AxiosError, we can try to extract the error message from the 
+  // response and display it in the pipeline or just use the Axios error message
+  if (axios.isAxiosError(err)) {
+
+    if (err.response) {
```

Comment:
Have you tried using the toJSON() function for AxiosErrors documented [here](https://axios-http.com/docs/handling_errors) ?

Reply:
`err.toJSON()` would convert the whole error object into a JSON which will contain the configuration and internals of the axios library. For me it is less readable than an error message with a stack trace. (X-Token is luckily masked, but I'm not sure about other secrets in the request or response body)